pinc/list_projects.inc

```diff
@@ -19,17 +19,14 @@ function list_projects_tersely( $where_condition, $order_clause, $limit_clause )
 
     while (list($title,$author,$language,$postednum) = mysqli_fetch_row($result))
     {
-        echo "<li><span class='title-listing'>";
+        echo "<li class='title-listing'>";
 
-        echo "<span class='book-title'>", '"', html_safe($title), '"', "</span>";
-        echo "&mdash;" . html_safe($author) . " ($language)";
+        echo "<span class='book-title'>" . html_safe($title) . "</span>";
+        echo ", " . html_safe($author) . " ($language)";
```

Comment:
Not that I anticipate it, but we may want to include html_safe on language if we don't validate it on insertion into db.

Reply:
That's a good idea. It certainly can't hurt!